addon/mixins/sidebar-navigation.js

```diff
@@ -16,14 +16,21 @@ export default Mixin.create({
 
   init() {
     this._super(...arguments);
+    this.updateSidebarMenuItems(undefined);
```

Comment:
You can also pass no arguments and it will function the same, and look a bit cleaner.